2024-12-24 00:39:10,827 [Thread-1] INFO  listener.ContextListener- contextClosed
2024-12-24 09:36:56,727 [main] INFO  util.DbWork- start migration
2024-12-24 09:36:57,056 [main] INFO  org.flywaydb.core.FlywayExecutor- Database: jdbc:postgresql://localhost:5432/sem_one (PostgreSQL 16.3)
2024-12-24 09:36:57,354 [main] INFO  org.flywaydb.core.internal.command.DbValidate- Successfully validated 2 migrations (execution time 00:00.164s)
2024-12-24 09:36:57,430 [main] INFO  org.flywaydb.core.internal.command.DbMigrate- Current version of schema "public": 2
2024-12-24 09:36:57,441 [main] INFO  org.flywaydb.core.internal.command.DbMigrate- Schema "public" is up to date. No migration necessary.
2024-12-24 09:36:57,469 [main] INFO  util.DbWork- migration done
2024-12-24 09:36:57,469 [main] INFO  listener.ContextListener- contextInitialized
2024-12-24 12:07:39,869 [http-nio-8080-exec-6] INFO  dao.NewsDao- Get the count of news - correct
2024-12-24 12:09:17,094 [http-nio-8080-exec-10] INFO  dao.NewsDao- Get the count of news - correct
2024-12-24 12:09:30,542 [Thread-1] INFO  listener.ContextListener- contextClosed
2024-12-24 12:10:49,029 [main] INFO  util.DbWork- start migration
2024-12-24 12:10:49,170 [main] INFO  org.flywaydb.core.FlywayExecutor- Database: jdbc:postgresql://localhost:5432/sem_one (PostgreSQL 16.3)
2024-12-24 12:10:49,297 [main] INFO  org.flywaydb.core.internal.command.DbValidate- Successfully validated 2 migrations (execution time 00:00.067s)
2024-12-24 12:10:49,330 [main] INFO  org.flywaydb.core.internal.command.DbMigrate- Current version of schema "public": 2
2024-12-24 12:10:49,336 [main] INFO  org.flywaydb.core.internal.command.DbMigrate- Schema "public" is up to date. No migration necessary.
2024-12-24 12:10:49,353 [main] INFO  util.DbWork- migration done
2024-12-24 12:10:49,353 [main] INFO  listener.ContextListener- contextInitialized
2024-12-24 12:10:52,502 [http-nio-8080-exec-1] INFO  dao.NewsDao- Get the count of news - correct
2024-12-24 12:10:58,212 [http-nio-8080-exec-7] INFO  dao.NewsDao- Get the count of news - correct
2024-12-24 13:26:28,404 [Thread-1] INFO  listener.ContextListener- contextClosed
2024-12-24 14:20:12,149 [main] INFO  util.DbWork- start migration
2024-12-24 14:20:12,303 [main] INFO  org.flywaydb.core.FlywayExecutor- Database: jdbc:postgresql://localhost:5432/sem_one (PostgreSQL 16.3)
2024-12-24 14:20:12,451 [main] INFO  org.flywaydb.core.internal.command.DbValidate- Successfully validated 2 migrations (execution time 00:00.090s)
2024-12-24 14:20:12,490 [main] INFO  org.flywaydb.core.internal.command.DbMigrate- Current version of schema "public": 2
2024-12-24 14:20:12,496 [main] INFO  org.flywaydb.core.internal.command.DbMigrate- Schema "public" is up to date. No migration necessary.
2024-12-24 14:20:12,515 [main] INFO  util.DbWork- migration done
2024-12-24 14:20:12,515 [main] INFO  listener.ContextListener- contextInitialized
2024-12-24 14:20:15,353 [http-nio-8080-exec-1] INFO  dao.NewsDao- Get the count of news - correct
2024-12-24 14:21:10,940 [Thread-1] INFO  listener.ContextListener- contextClosed
2024-12-24 14:21:40,483 [main] INFO  util.DbWork- start migration
2024-12-24 14:21:40,619 [main] INFO  org.flywaydb.core.FlywayExecutor- Database: jdbc:postgresql://localhost:5432/sem_one (PostgreSQL 16.3)
2024-12-24 14:21:40,755 [main] INFO  org.flywaydb.core.internal.command.DbValidate- Successfully validated 2 migrations (execution time 00:00.078s)
2024-12-24 14:21:40,788 [main] INFO  org.flywaydb.core.internal.command.DbMigrate- Current version of schema "public": 2
2024-12-24 14:21:40,795 [main] INFO  org.flywaydb.core.internal.command.DbMigrate- Schema "public" is up to date. No migration necessary.
2024-12-24 14:21:40,813 [main] INFO  util.DbWork- migration done
2024-12-24 14:21:40,813 [main] INFO  listener.ContextListener- contextInitialized
2024-12-24 14:24:46,640 [Thread-1] INFO  listener.ContextListener- contextClosed
2024-12-24 14:27:32,388 [main] INFO  util.DbWork- start migration
2024-12-24 14:27:32,557 [main] INFO  org.flywaydb.core.FlywayExecutor- Database: jdbc:postgresql://localhost:5432/sem_one (PostgreSQL 16.3)
2024-12-24 14:27:32,708 [main] ERROR util.DbWork- org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Migration checksum mismatch for migration version 1
-> Applied to database : 824496378
-> Resolved locally    : 1314710648
Either revert the changes to the migration, or run repair to update the schema history.
Need more flexibility with validation rules? Learn more: https://rd.gt/3AbJUZE
2024-12-24 14:27:32,708 [main] INFO  listener.ContextListener- util.DbException: Cannot connect to DB
2024-12-24 14:27:32,819 [main] INFO  util.DbWork- start migration
2024-12-24 14:27:32,835 [main] INFO  org.flywaydb.core.FlywayExecutor- Database: jdbc:postgresql://localhost:5432/sem_one (PostgreSQL 16.3)
2024-12-24 14:27:32,865 [main] ERROR util.DbWork- org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Migration checksum mismatch for migration version 1
-> Applied to database : 824496378
-> Resolved locally    : 1314710648
Either revert the changes to the migration, or run repair to update the schema history.
Need more flexibility with validation rules? Learn more: https://rd.gt/3AbJUZE
2024-12-24 14:27:32,865 [main] INFO  listener.ContextListener- util.DbException: Cannot connect to DB
2024-12-24 14:28:20,680 [main] INFO  util.DbWork- start migration
2024-12-24 14:28:20,820 [main] INFO  org.flywaydb.core.FlywayExecutor- Database: jdbc:postgresql://localhost:5432/sem_one (PostgreSQL 16.3)
2024-12-24 14:28:20,973 [main] ERROR util.DbWork- org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Migration checksum mismatch for migration version 1
-> Applied to database : 824496378
-> Resolved locally    : 1314710648
Either revert the changes to the migration, or run repair to update the schema history.
Need more flexibility with validation rules? Learn more: https://rd.gt/3AbJUZE
2024-12-24 14:28:20,974 [main] INFO  listener.ContextListener- util.DbException: Cannot connect to DB
2024-12-24 14:28:21,081 [main] INFO  util.DbWork- start migration
2024-12-24 14:28:21,098 [main] INFO  org.flywaydb.core.FlywayExecutor- Database: jdbc:postgresql://localhost:5432/sem_one (PostgreSQL 16.3)
2024-12-24 14:28:21,124 [main] ERROR util.DbWork- org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Migration checksum mismatch for migration version 1
-> Applied to database : 824496378
-> Resolved locally    : 1314710648
Either revert the changes to the migration, or run repair to update the schema history.
Need more flexibility with validation rules? Learn more: https://rd.gt/3AbJUZE
2024-12-24 14:28:21,124 [main] INFO  listener.ContextListener- util.DbException: Cannot connect to DB
2024-12-24 14:31:49,580 [main] INFO  util.DbWork- start migration
2024-12-24 14:31:49,725 [main] INFO  org.flywaydb.core.FlywayExecutor- Database: jdbc:postgresql://localhost:5432/sem_one (PostgreSQL 16.3)
2024-12-24 14:31:49,902 [main] ERROR util.DbWork- org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Migration checksum mismatch for migration version 1
-> Applied to database : 824496378
-> Resolved locally    : 1314710648
Either revert the changes to the migration, or run repair to update the schema history.
Need more flexibility with validation rules? Learn more: https://rd.gt/3AbJUZE
2024-12-24 14:31:49,902 [main] INFO  listener.ContextListener- util.DbException: Cannot connect to DB
2024-12-24 14:31:50,013 [main] INFO  util.DbWork- start migration
2024-12-24 14:31:50,028 [main] INFO  org.flywaydb.core.FlywayExecutor- Database: jdbc:postgresql://localhost:5432/sem_one (PostgreSQL 16.3)
2024-12-24 14:31:50,056 [main] ERROR util.DbWork- org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Migration checksum mismatch for migration version 1
-> Applied to database : 824496378
-> Resolved locally    : 1314710648
Either revert the changes to the migration, or run repair to update the schema history.
Need more flexibility with validation rules? Learn more: https://rd.gt/3AbJUZE
2024-12-24 14:31:50,056 [main] INFO  listener.ContextListener- util.DbException: Cannot connect to DB
2024-12-24 14:32:38,085 [main] INFO  util.DbWork- start migration
2024-12-24 14:32:38,264 [main] INFO  org.flywaydb.core.FlywayExecutor- Database: jdbc:postgresql://localhost:5432/sem_one (PostgreSQL 16.3)
2024-12-24 14:32:38,432 [main] ERROR util.DbWork- org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Migration checksum mismatch for migration version 1
-> Applied to database : 824496378
-> Resolved locally    : 1314710648
Either revert the changes to the migration, or run repair to update the schema history.
Need more flexibility with validation rules? Learn more: https://rd.gt/3AbJUZE
2024-12-24 14:32:38,433 [main] INFO  listener.ContextListener- util.DbException: Cannot connect to DB
2024-12-24 14:32:38,649 [main] INFO  util.DbWork- start migration
2024-12-24 14:32:38,672 [main] INFO  org.flywaydb.core.FlywayExecutor- Database: jdbc:postgresql://localhost:5432/sem_one (PostgreSQL 16.3)
2024-12-24 14:32:38,699 [main] ERROR util.DbWork- org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Migration checksum mismatch for migration version 1
-> Applied to database : 824496378
-> Resolved locally    : 1314710648
Either revert the changes to the migration, or run repair to update the schema history.
Need more flexibility with validation rules? Learn more: https://rd.gt/3AbJUZE
2024-12-24 14:32:38,699 [main] INFO  listener.ContextListener- util.DbException: Cannot connect to DB
2024-12-24 14:34:02,869 [main] INFO  util.DbWork- start migration
2024-12-24 14:34:03,062 [main] INFO  org.flywaydb.core.FlywayExecutor- Database: jdbc:postgresql://localhost:5432/sem_one (PostgreSQL 16.3)
2024-12-24 14:34:03,225 [main] ERROR util.DbWork- org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Migration checksum mismatch for migration version 1
-> Applied to database : 824496378
-> Resolved locally    : 1314710648
Either revert the changes to the migration, or run repair to update the schema history.
Need more flexibility with validation rules? Learn more: https://rd.gt/3AbJUZE
2024-12-24 14:34:03,225 [main] INFO  listener.ContextListener- util.DbException: Cannot connect to DB
2024-12-24 14:34:03,338 [main] INFO  util.DbWork- start migration
2024-12-24 14:34:03,360 [main] INFO  org.flywaydb.core.FlywayExecutor- Database: jdbc:postgresql://localhost:5432/sem_one (PostgreSQL 16.3)
2024-12-24 14:34:03,390 [main] ERROR util.DbWork- org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Migration checksum mismatch for migration version 1
-> Applied to database : 824496378
-> Resolved locally    : 1314710648
Either revert the changes to the migration, or run repair to update the schema history.
Need more flexibility with validation rules? Learn more: https://rd.gt/3AbJUZE
2024-12-24 14:34:03,390 [main] INFO  listener.ContextListener- util.DbException: Cannot connect to DB
2024-12-24 14:37:37,862 [main] INFO  util.DbWork- start migration
2024-12-24 14:37:37,994 [main] INFO  org.flywaydb.core.FlywayExecutor- Database: jdbc:postgresql://localhost:5432/sem_one (PostgreSQL 16.3)
2024-12-24 14:37:38,156 [main] ERROR util.DbWork- org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Migration checksum mismatch for migration version 1
-> Applied to database : 824496378
-> Resolved locally    : 1314710648
Either revert the changes to the migration, or run repair to update the schema history.
Need more flexibility with validation rules? Learn more: https://rd.gt/3AbJUZE
2024-12-24 14:37:38,156 [main] INFO  listener.ContextListener- util.DbException: Cannot connect to DB
2024-12-24 14:37:38,276 [main] INFO  util.DbWork- start migration
2024-12-24 14:37:38,292 [main] INFO  org.flywaydb.core.FlywayExecutor- Database: jdbc:postgresql://localhost:5432/sem_one (PostgreSQL 16.3)
2024-12-24 14:37:38,318 [main] ERROR util.DbWork- org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Migration checksum mismatch for migration version 1
-> Applied to database : 824496378
-> Resolved locally    : 1314710648
Either revert the changes to the migration, or run repair to update the schema history.
Need more flexibility with validation rules? Learn more: https://rd.gt/3AbJUZE
2024-12-24 14:37:38,318 [main] INFO  listener.ContextListener- util.DbException: Cannot connect to DB
2024-12-24 14:40:05,354 [main] INFO  util.DbWork- start migration
2024-12-24 14:40:05,500 [main] INFO  org.flywaydb.core.FlywayExecutor- Database: jdbc:postgresql://localhost:5432/sem_one (PostgreSQL 16.3)
2024-12-24 14:40:05,649 [main] ERROR util.DbWork- org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Migration checksum mismatch for migration version 1
-> Applied to database : 824496378
-> Resolved locally    : 1314710648
Either revert the changes to the migration, or run repair to update the schema history.
Need more flexibility with validation rules? Learn more: https://rd.gt/3AbJUZE
2024-12-24 14:40:05,649 [main] INFO  listener.ContextListener- util.DbException: Cannot connect to DB
2024-12-24 14:40:05,775 [main] INFO  util.DbWork- start migration
2024-12-24 14:40:05,791 [main] INFO  org.flywaydb.core.FlywayExecutor- Database: jdbc:postgresql://localhost:5432/sem_one (PostgreSQL 16.3)
2024-12-24 14:40:05,818 [main] ERROR util.DbWork- org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Migration checksum mismatch for migration version 1
-> Applied to database : 824496378
-> Resolved locally    : 1314710648
Either revert the changes to the migration, or run repair to update the schema history.
Need more flexibility with validation rules? Learn more: https://rd.gt/3AbJUZE
2024-12-24 14:40:05,819 [main] INFO  listener.ContextListener- util.DbException: Cannot connect to DB
2024-12-24 14:41:37,006 [main] INFO  util.DbWork- start migration
2024-12-24 14:41:37,144 [main] INFO  org.flywaydb.core.FlywayExecutor- Database: jdbc:postgresql://localhost:5432/sem_one (PostgreSQL 16.3)
2024-12-24 14:41:37,300 [main] ERROR util.DbWork- org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Migration checksum mismatch for migration version 1
-> Applied to database : 824496378
-> Resolved locally    : 1314710648
Either revert the changes to the migration, or run repair to update the schema history.
Need more flexibility with validation rules? Learn more: https://rd.gt/3AbJUZE
2024-12-24 14:41:37,300 [main] INFO  listener.ContextListener- util.DbException: Cannot connect to DB
2024-12-24 14:41:37,407 [main] INFO  util.DbWork- start migration
2024-12-24 14:41:37,420 [main] INFO  org.flywaydb.core.FlywayExecutor- Database: jdbc:postgresql://localhost:5432/sem_one (PostgreSQL 16.3)
2024-12-24 14:41:37,449 [main] ERROR util.DbWork- org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Migration checksum mismatch for migration version 1
-> Applied to database : 824496378
-> Resolved locally    : 1314710648
Either revert the changes to the migration, or run repair to update the schema history.
Need more flexibility with validation rules? Learn more: https://rd.gt/3AbJUZE
2024-12-24 14:41:37,449 [main] INFO  listener.ContextListener- util.DbException: Cannot connect to DB
2024-12-24 14:43:13,964 [main] INFO  util.DbWork- start migration
2024-12-24 14:43:14,112 [main] INFO  org.flywaydb.core.FlywayExecutor- Database: jdbc:postgresql://localhost:5432/sem_one (PostgreSQL 16.3)
2024-12-24 14:43:14,268 [main] ERROR util.DbWork- org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Migration checksum mismatch for migration version 1
-> Applied to database : 824496378
-> Resolved locally    : 1314710648
Either revert the changes to the migration, or run repair to update the schema history.
Need more flexibility with validation rules? Learn more: https://rd.gt/3AbJUZE
2024-12-24 14:43:14,269 [main] INFO  listener.ContextListener- util.DbException: Cannot connect to DB
2024-12-24 14:43:14,370 [main] INFO  util.DbWork- start migration
2024-12-24 14:43:14,385 [main] INFO  org.flywaydb.core.FlywayExecutor- Database: jdbc:postgresql://localhost:5432/sem_one (PostgreSQL 16.3)
2024-12-24 14:43:14,411 [main] ERROR util.DbWork- org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Migration checksum mismatch for migration version 1
-> Applied to database : 824496378
-> Resolved locally    : 1314710648
Either revert the changes to the migration, or run repair to update the schema history.
Need more flexibility with validation rules? Learn more: https://rd.gt/3AbJUZE
2024-12-24 14:43:14,411 [main] INFO  listener.ContextListener- util.DbException: Cannot connect to DB
2024-12-24 14:50:16,999 [main] INFO  util.DbWork- start migration
2024-12-24 14:50:17,152 [main] INFO  org.flywaydb.core.FlywayExecutor- Database: jdbc:postgresql://localhost:5432/sem_one (PostgreSQL 16.3)
2024-12-24 14:50:17,330 [main] ERROR util.DbWork- org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Migration checksum mismatch for migration version 1
-> Applied to database : 824496378
-> Resolved locally    : 1314710648
Either revert the changes to the migration, or run repair to update the schema history.
Need more flexibility with validation rules? Learn more: https://rd.gt/3AbJUZE
2024-12-24 14:50:17,331 [main] INFO  listener.ContextListener- util.DbException: Cannot connect to DB
2024-12-24 14:50:17,527 [main] INFO  util.DbWork- start migration
2024-12-24 14:50:17,544 [main] INFO  org.flywaydb.core.FlywayExecutor- Database: jdbc:postgresql://localhost:5432/sem_one (PostgreSQL 16.3)
2024-12-24 14:50:17,573 [main] ERROR util.DbWork- org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Migration checksum mismatch for migration version 1
-> Applied to database : 824496378
-> Resolved locally    : 1314710648
Either revert the changes to the migration, or run repair to update the schema history.
Need more flexibility with validation rules? Learn more: https://rd.gt/3AbJUZE
2024-12-24 14:50:17,574 [main] INFO  listener.ContextListener- util.DbException: Cannot connect to DB
2024-12-24 14:53:24,225 [main] INFO  util.DbWork- start migration
2024-12-24 14:53:24,377 [main] INFO  org.flywaydb.core.FlywayExecutor- Database: jdbc:postgresql://localhost:5432/sem_one (PostgreSQL 16.3)
2024-12-24 14:53:24,540 [main] ERROR util.DbWork- org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Migration checksum mismatch for migration version 1
-> Applied to database : 824496378
-> Resolved locally    : 1314710648
Either revert the changes to the migration, or run repair to update the schema history.
Need more flexibility with validation rules? Learn more: https://rd.gt/3AbJUZE
2024-12-24 14:53:24,540 [main] INFO  listener.ContextListener- util.DbException: Cannot connect to DB
2024-12-24 14:53:24,660 [main] INFO  util.DbWork- start migration
2024-12-24 14:53:24,676 [main] INFO  org.flywaydb.core.FlywayExecutor- Database: jdbc:postgresql://localhost:5432/sem_one (PostgreSQL 16.3)
2024-12-24 14:53:24,702 [main] ERROR util.DbWork- org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Migration checksum mismatch for migration version 1
-> Applied to database : 824496378
-> Resolved locally    : 1314710648
Either revert the changes to the migration, or run repair to update the schema history.
Need more flexibility with validation rules? Learn more: https://rd.gt/3AbJUZE
2024-12-24 14:53:24,703 [main] INFO  listener.ContextListener- util.DbException: Cannot connect to DB
2024-12-24 14:56:55,120 [main] INFO  util.DbWork- start migration
2024-12-24 14:56:55,246 [main] INFO  org.flywaydb.core.FlywayExecutor- Database: jdbc:postgresql://localhost:5432/sem_one (PostgreSQL 16.3)
2024-12-24 14:56:55,396 [main] ERROR util.DbWork- org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Migration checksum mismatch for migration version 1
-> Applied to database : 824496378
-> Resolved locally    : 1314710648
Either revert the changes to the migration, or run repair to update the schema history.
Need more flexibility with validation rules? Learn more: https://rd.gt/3AbJUZE
2024-12-24 14:56:55,397 [main] INFO  listener.ContextListener- util.DbException: Cannot connect to DB
2024-12-24 14:56:55,502 [main] INFO  util.DbWork- start migration
2024-12-24 14:56:55,517 [main] INFO  org.flywaydb.core.FlywayExecutor- Database: jdbc:postgresql://localhost:5432/sem_one (PostgreSQL 16.3)
2024-12-24 14:56:55,545 [main] ERROR util.DbWork- org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Migration checksum mismatch for migration version 1
-> Applied to database : 824496378
-> Resolved locally    : 1314710648
Either revert the changes to the migration, or run repair to update the schema history.
Need more flexibility with validation rules? Learn more: https://rd.gt/3AbJUZE
2024-12-24 14:56:55,545 [main] INFO  listener.ContextListener- util.DbException: Cannot connect to DB
2024-12-24 14:58:48,491 [main] INFO  util.DbWork- start migration
2024-12-24 14:58:48,619 [main] INFO  org.flywaydb.core.FlywayExecutor- Database: jdbc:postgresql://localhost:5432/sem_one (PostgreSQL 16.3)
2024-12-24 14:58:48,747 [main] INFO  org.flywaydb.core.internal.command.DbValidate- Successfully validated 2 migrations (execution time 00:00.075s)
2024-12-24 14:58:48,780 [main] INFO  org.flywaydb.core.internal.command.DbMigrate- Current version of schema "public": 2
2024-12-24 14:58:48,785 [main] INFO  org.flywaydb.core.internal.command.DbMigrate- Schema "public" is up to date. No migration necessary.
2024-12-24 14:58:48,803 [main] INFO  util.DbWork- migration done
2024-12-24 14:58:48,803 [main] INFO  listener.ContextListener- contextInitialized
2024-12-24 14:58:51,515 [http-nio-8080-exec-2] INFO  dao.NewsDao- Get the count of news - correct
2024-12-24 14:59:07,478 [Thread-1] INFO  listener.ContextListener- contextClosed
